### PR TITLE
Make Github recognize the LICENSE

### PR DESCRIPTION
**Pull request by Skeen**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,27 +1,25 @@
-Copyright (c) Django Software Foundation and individual contributors.
+Copyright (c) 2001-2017 Django Software Foundation and individual contributors.
```
Comment (timgraham) [inline LICENSE]: Is it required to include the date? I find that annoying to update each year, though we already have to do it for the Python license.

Comment (Skeen) [reply LICENSE]: The answer is no. It is not required to make GitHub recognize the license.
- It is however included in the default BSD-3-Clause license text.

Comment (Skeen) [reply LICENSE]: I have removed the year from the License.

**Comment by Skeen** [comment]
@timgraham: What would you consider a meaningful name for the `LICENSE.python` file, if it cannot be `LICENSE.*`?

**Comment by timgraham** [comment]
I guess LICENSE-python would be fine.

File: LICENSE [hunk 2]
```diff
@@ -4,24 +4,22 @@ All rights reserved.
...
-
-    3. Neither the name of Django nor the names of its contributors may be used
-       to endorse or promote products derived from this software without
-       specific prior written permission.
+ * Redistributions of source code must retain the above copyright notice, this
+   list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright notice,
+   this list of conditions and the following disclaimer in the documentation
+   and/or other materials provided with the distribution.
+ * Neither the name of the Pika project nor the names of its contributors may be used
```
Comment (jdufresne) [inline LICENSE]: No comment on the overall change, but I see a reference to the Pika project here. I believe this was an unintentional copy and paste mistake.

Comment (Skeen) [reply LICENSE]: Yup, I'm a moron.

File: LICENSE [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
-       notice, this list of conditions and the following disclaimer in the
-       documentation and/or other materials provided with the distribution.
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
 
-    3. Neither the name of Django nor the names of its contributors may be used
-       to endorse or promote products derived from this software without
-       specific prior written permission.
+* Neither the name of the copyright holder nor the names of its
```
Comment (timgraham) [inline LICENSE]: Is the line wrapping copied from some place and the existing line wrapping doesn't work? Here, for example, it seems that "contributors" would fit on this line based on the wrapping of other lines.

Comment (Skeen) [reply LICENSE]: I agree that it would fit. The licence file I copied in is the one GitHub provides for new projects.

**Comment by Skeen** [comment]
Using LICENSE_python, seems to interfere with GitHub picking up the LICENSE file.
I'm not sure why though, as it does not seem to hit any of the regexes of the ruby gem GitHub uses.

**Comment by atombrella** [comment]
I think BSD is not a recognized option due to MIT being BSD being quite similar. Instead of changing file names (which wouldn't have any effect), https://choosealicense.com/ are valid options if Github should recognize `LICENSE` and show the highlights from `choosealicense`.

**Comment by timgraham** [comment]
@Skeen, do you want to continue this or should we close it?

**Pull request closed without merging**